api/src/main/resources/emails/unpublish_workspace/content.html

```diff
@@ -29,7 +29,7 @@
 <div>We are writing to you on behalf of the ${ALL_OF_US} Research Program’s Researcher Workbench
   Support team. A workspace you own titled
   <span class="workspace">'${WORKSPACE_NAME}' (${WORKSPACE_NAMESPACE})</span> has been
-  un-published  by ${ALL_OF_US} Support team</div>
+  un-published by the ${ALL_OF_US} Support team.</div>
```

Comment:
Do we need first name/last name for unpublished email?

Reply:
good catch!